src/test/chainstate_write_tests.cpp

```diff
@@ -42,4 +42,58 @@ BOOST_FIXTURE_TEST_CASE(chainstate_write_interval, TestingSetup)
     BOOST_CHECK(sub->m_did_flush);
 }
 
+// Test that we do PERIODIC flushes inside ActivateBestChain.
+// This is necessary for reindex-chainstate to be able to periodically flush
+// before reaching chain tip.
+BOOST_FIXTURE_TEST_CASE(write_during_multiblock_activation, TestChain100Setup)
+{
+    struct TestSubscriber final : CValidationInterface {
```

Comment:
Nit - [developer notes suggests](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md?plain=1#L70-L71) to put the braces to new lines:
```md
  - Braces on new lines for classes, functions, methods.
```

Reply:
What about structs?

Reply:
`AfterStruct` seems to fall back to `AfterClass` if unspecified https://github.com/bitcoin/bitcoin/blob/master/src/.clang-format#L19

Reply:
Done.